app/src/org/commcare/android/javarosa/IntentCallout.java

```diff
@@ -319,6 +316,7 @@ private void storePointerToFileResponse(TreeReference ref, String responseValue,
         } catch (IOException ioe) {
             ioe.printStackTrace();
             Log.e(TAG,"CommCare failed to copy a binary input from an intent callout from: " + responseValue);
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/dimagi/commcare-android/blob/efa4e6d4807709ea34abb41e48e4b6bd8249726c/app/src/org/commcare/android/javarosa/IntentCallout.java#L318"></span>Using 'Log' instead of 'Timber'</td>
    </tr>
  </tbody>
</table>